docker/README.md

```diff
@@ -94,11 +94,29 @@ or if you wish to use the `clang` compiler with `openmp` offloading:
 docker build --build-arg base=devitocodes/base:nvidia-clang --network=host --file docker/Dockerfile --tag devito .
 ```
 
-and finally for AMD architectures:
+and finally, for AMD architectures:
 
 ```bash
 docker build --build-arg base=devitocodes/base:amd --network=host --file docker/Dockerfile --tag devito .
 ```
 
 
-[Devito]:https://github.com/devitocodes/devito
\ No newline at end of file
+## Build/Debug a base image (Useful for development only)
+
+This section is primarily helpful in contributing to or debugging the docker images.
+Devito images are built on top of base layers that provide the necessary software stack (compilers, utilities etc.).
+
+To build the base image yourself locally, you need to run the standard build command using the provided Dockerfile.
+
+```bash
+# To locally build the base CPU image for gcc, run: 
+docker build . --file docker/Dockerfile.cpu --tag devito-gcc --build-arg arch=gcc
+# To locally build the Devito image using the previously built base, run: 
+docker build . --file docker/Dockerfile.devito --tag devito_img --build-arg base=devito-gcc:latest
```

Comment:
I don't understand how is this new section useful... aren't these commands basically the same as in those in the section(s) above?

Reply:
No, above, we do not build the base images.

Reply:
ok

but the two commands below to run tests are redundant, at least, aren't they?

Reply:
Why redundant? They are optional commands so that the developer tests that everything is ok